### PR TITLE
Update links and diagrams in KEB

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Fix the links to the old navigation
- Update diagrams according to the [new guidelines](https://kyma-project.io/community/guidelines/content-guidelines/08-diagrams/)
- Fix formatting

**Related issue(s)**
(https://github.com/kyma-project/control-plane/issues/676)

### CODE REVIEW

File: docs/kyma-environment-broker/03-08-kyma-versions.md [hunk 1]
```diff
@@ -53,4 +50,4 @@ You can also specify a Kyma version using the **kymaVersion** provisioning param
 The **kymaVersion** provisioning parameter overrides the default settings.
 To enable this feature, set the **APP_ENABLE_ON_DEMAND_VERSION** environment variable to `true`.
 
-**NOTE:** The default Kyma version for the preview plan is set separately in the **APP_KYMA_PREVIEW_VERSION** environment variable.
\ No newline at end of file
+**NOTE:** The default Kyma version for the preview plan is set separately in the **APP_KYMA_PREVIEW_VERSION** environment variable.
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/03-08-kyma-versions.md:53]:
```suggestion
>**NOTE:** The default Kyma version for the preview plan is set separately in the **APP_KYMA_PREVIEW_VERSION** environment variable.
```

File: docs/kyma-environment-broker/03-10-orchestration.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Orchestration
 type: Details
 ---
 
-Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [follow this tutorial](#tutorials-orchestrate-kyma-upgrade). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](#details-runtime-operations) logic on the selected Runtimes.
+Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [follow this tutorial](08-05-orchestrate-kyma-upgrade.md). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](03-03-runtime-operations.md) logic on the selected Runtimes.
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/03-10-orchestration.md:6]:
```suggestion
Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [follow this tutorial](08-05-orchestrate-kyma-upgrade.md). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](03-03-runtime-operations.md#upgrade) logic on the selected Runtimes.
```

Should the metadata in the doc stay?

File: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md [hunk 3]
```diff
@@ -25,13 +22,13 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
 
    > **NOTE:** INSTANCE_ID and NAME must be unique. It is recommended to use UUID as an INSTANCE_ID.
 
-2. Get the [access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
+2. Get the [access token](03-05-authorization.md). Export this variable based on the token you got from the OAuth client:
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md:25]:
```suggestion
2. Get the [access token](03-05-authorization.md#get-the-access-token). Export this variable based on the token you got from the OAuth client:
```

File: docs/kyma-environment-broker/08-02-deprovisioning-kyma-environment.md [hunk 4]
```diff
@@ -14,7 +11,7 @@ This tutorial shows how to deprovision Kyma Runtime on Azure using Kyma Environm
    export INSTANCE_ID={INSTANCE_ID_FROM_PROVISIONING_CALL}
    ```
 
-2. Get the [access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
+2. Get the [access token](03-05-authorization.md). Export this variable based on the token you got from the OAuth client:
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/08-02-deprovisioning-kyma-environment.md:14]:
```suggestion
2. Get the [access token](03-05-authorization.md#get-the-access-token). Export this variable based on the token you got from the OAuth client:
```

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 5]
```diff
@@ -53,7 +50,7 @@ This tutorial shows how to check the orchestration status. Using this API, you c
 
 ## Fetch all orchestrations status
 
-Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-orchestration) to verify that the orchestration succeeded.
+Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-10-orchestration.md) to verify that the orchestration succeeded.
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:53]: I cannot see any request header details in the linked document. Did you mean `03-05-authorization.md`?

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 6]
```diff
@@ -98,7 +95,7 @@ A successful call returns the list of all orchestrations:
    export ORCHESTRATION_ID={OBTAINED_ORCHESTRATION_ID}
    ```
 
-2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-orchestration) to fetch the list of the upgrade operations for a given orchestration.
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-10-orchestration.md) to fetch the list of the upgrade operations for a given orchestration.
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:98]: I cannot see any request header details in the linked document. Did you mean `03-05-authorization.md`?

File: docs/kyma-environment-broker/08-06-orchestration-status.md [hunk 7]
```diff
@@ -150,7 +147,7 @@ A successful call returns the list of all orchestrations:
    export OPERATION_ID={SCHEDULED_OPERATION_ID}
    ```
 
-2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-orchestration) to fetch the details for given operation.
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-10-orchestration.md) to fetch the details for given operation.
```
Comment (mmitoraj) [inline docs/kyma-environment-broker/08-06-orchestration-status.md:150]: I cannot see any request header details in the linked document. Did you mean `03-05-authorization.md`?

**Review by mmitoraj — COMMENTED** [review]
I added a few suggestions :)

File: docs/kyma-environment-broker/03-10-orchestration.md (hunk 2 above)
Comment (klaudiagrz) [reply docs/kyma-environment-broker/03-10-orchestration.md]: No, you're right, I'll change it.

File: docs/kyma-environment-broker/08-06-orchestration-status.md (hunk 5 above)
Comment (klaudiagrz) [reply docs/kyma-environment-broker/08-06-orchestration-status.md:53]: Yes, wrong link. Thanks for spotting! 🙂

**Review by mmitoraj — APPROVED** [review]

**Review by szwedm — APPROVED** [review]

**Pull request merged**